sourced/ml/algorithms/uast_ids_to_bag.py

```diff
@@ -4,6 +4,8 @@
 
 from sourced.ml.algorithms import TokenParser, NoopTokenParser
 from sourced.ml.algorithms.uast_to_bag import Uast2BagBase
+from sourced.ml.utils import IDENTIFIER, QUALIFIED
+from sourced.ml.utils import uast2sequence
```

Comment:
`uast2sequence` should be somewhere in "algorithms"

Reply:
Will move it